src/hooks/useFetchCollection.ts

```diff
@@ -13,36 +13,42 @@ const assignTypes = <T extends object>() => {
   };
 };
 
-export const useFetchData = <T extends object>(collectionName: string) => {
+export const useFetchCollection = <T extends object>(
+  collectionName: string
+) => {
   const [data, setData] = useState<T[] | null>(null);
   const [isLoading, setIsLoading] = useState(false);
   const [error, setError] = useState<string | null>(null);
 
   useEffect(() => {
     setIsLoading(true);
-    // najit ve firebase document
+
     const userRef = collection(db, collectionName).withConverter(
       assignTypes<T>()
     );
     const unsubscribe = onSnapshot(
       userRef,
       (snapshot) => {
-        if (snapshot) {
+        if (!snapshot.empty) {
```

Comment:
jses si 100%, ze je to takhle spravne, ze ten snapshot ma empty? Ja si jisty porpavde nejsem. Prosim o kontrolu, mel jsem dojem, ze snapshot bez empty stacil.